docker-compose.router.yml

```diff
@@ -1,24 +1,16 @@
 version: '3'
 services:
   apollo-router:
-    container_name: apollo-router
-    image: ghcr.io/apollographql/router:v1.3.0
+    container_name: events-graphl-router
```

Comment:
Again missing this guy. 
![image](https://user-images.githubusercontent.com/1314604/202647050-4618d624-b1b0-4458-8ef1-1193366ddf7f.png)

Reply:
did the typo with search&replace, so I'll fix it with that too and we get the Q back